cpp/Brewfile

```diff
@@ -25,7 +25,6 @@ brew "flatbuffers"
 brew "git"
 brew "glog"
 brew "grpc"
-brew "llvm"
```

Comment:
Can we revert this change?

Reply:
Do we actually want brew `llvm` and `llvm@8`? Reverting anyways.

Reply:
`llvm@8` is just for linting. We use `clang-format` and so on.
`llvm` is for Gandiva.